src/components/TradeWidget/index.tsx

```diff
@@ -184,7 +190,7 @@ const TradeWidget: React.FC = () => {
       })
       if (success) {
         // reset form on successful order placing
-        reset()
+        reset(DEFAULT_FORM_STATE)
```

Comment:
Should this also go into `L102::defaultValues`

Reply:
Don't think so, because those default values are used to set the form values from the url.
Resetting it here should be enough.

Reply:
Still, something is wrong with resetting. Unlimited checkbox gets stuck.